src/telemetryReporter.ts

```diff
@@ -22,26 +98,39 @@ export default class TelemetryReporter {
 		this._telemetryReporter = new VsCodeTelemetryReporter(extensionId, extensionVersion, key);
 	}
 
+	/**
+	 * Creates a View event that can be sent later. This is used to log that a particular page or item was seen.
+	 * @param view The name of the page or item that was viewed
+	 */
+	public createViewEvent(view: string): TelemetryEvent {
+		return new TelemetryEventImpl(this._telemetryReporter, 'view', {
```

Comment:
would it be better to have `view`, `action` etc. as an enum?

Reply:
The actual event name? It's only used once and doesn't need to be referenced anywhere else so I'd rather just leave it as-is. No reason to add more clutter when it wouldn't provide any value.